packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -689,7 +689,18 @@
  
    ## Reference to one or more secrets to be used when pulling images
    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
-@@ -364,25 +1010,77 @@
+@@ -234,6 +880,10 @@
+   ## Deploys the windows-exporter and Windows-specific dashboards and rules (job name must be 'windows-exporter')
+   enabled: false
+ 
++loggingMonitoring:
```

Comment:
since metrics are enables seperately for fluentd and fluentbit are enabled separately in rancher-logging, we should do the same here